src/crimson/osd/object_context_loader.h

```diff
@@ -39,6 +39,13 @@ class ObjectContextLoader {
   load_obc_iertr::future<> with_clone_obc(hobject_t oid,
                                           with_obc_func_t&& func);
 
+  // Use this variant in the case where the head object
+  // obc is already locked. Avoid nesting
+  // with_head_obc() as in using with_clone_obc().
+  load_obc_iertr::future<> with_clone_obc_only(const SnapSet& snap_set,
```

Comment:
Is there a reasonable way to rebuild the regular `with_clone_obc()` upon this bit?